server/sonar-server/src/main/java/org/sonar/server/component/ComponentCleanerService.java

```diff
@@ -29,27 +29,27 @@
 import org.sonar.db.DbSession;
 import org.sonar.db.MyBatis;
 import org.sonar.db.component.ComponentDto;
-import org.sonar.server.issue.index.IssueAuthorizationIndexer;
-import org.sonar.server.issue.index.IssueIndexer;
 import org.sonar.server.component.es.ProjectMeasuresIndexer;
+import org.sonar.server.issue.index.IssueIndexer;
+import org.sonar.server.permission.index.PermissionIndexer;
 import org.sonar.server.test.index.TestIndexer;
 
 @ServerSide
 @ComputeEngineSide
 public class ComponentCleanerService {
 
   private final DbClient dbClient;
-  private final IssueAuthorizationIndexer issueAuthorizationIndexer;
+  private final PermissionIndexer permissionIndexer;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "permissionIndexer" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)